src/main/spec/internal-metering-api-spec.yaml

```diff
@@ -63,7 +76,7 @@ paths:
             type: integer
             minimum: 0
             description: "Defines the amount of time (in minutes) that will be used to calculate a metric query's start date based on a given end date.
-For example, given an end date of 2021-01-06T00:00:00Z and a rangeInMinutes of 60, the calculated start date will be: 2021-01-05T23:00:00Z.  Default (60) is store in application-metering-job.yaml"
+For example, given an end date of 2021-01-06T00:00:00Z and a rangeInMinutes of 60, the calculated start date will be: 2021-01-05T23:00:00Z.  Default (60) is store in application-openshift-metering-worker.yaml"
```

Comment:
in application-openshift-metering-worker.yaml and application-metrics-rhel.yaml

Reply:
updated